clients/common.go

```diff
@@ -16,6 +16,7 @@ import (
 	"net/url"
 
 	"gopkg.in/src-d/go-git.v3/clients/common"
+	"gopkg.in/src-d/go-git.v3/clients/file"
```

Comment:
Altough, it is not really a client as there is no server, but that is a different problem we must solve :).